### PR TITLE
Update ppp for xiongjya.md

### PR DESCRIPTION
**Pull request by xiongjya**

### CODE REVIEW

File: docs/team/xiongjya.md [hunk 1]
```diff
@@ -14,35 +14,52 @@ and it still has the benefits of a Graphical User Interface (GUI) created with J
...
+
+
+
+* **New Feature**: Added the ability to add remarks to elderly contacts.
+    * What it does: Allows the user to add additional notes in the form of a remark, to each elderly contact that has been added into NurseyBook.
+    * Justification: Instead of using other applications/pen or paper to take down additional notes for each elderly, the user can keep track of such information within the application itself. There will be no need for the user to switch between different applications.
+
+
+* **New Feature**: Sort newly added tasks into NurseyBook in chronological order.
+    * What it does: When the user adds a task into NurseyBook, the task will not be added to the end of the list. Instead, it will be slotted into a position in the task list, to make sure that the task list displays tasks in a chronological order.
```
Comment (nicole-luo-exe) [inline docs/team/xiongjya.md:27]: How about "when a user adds a task into NurseyBook, the task is positioned in the list according to its time such that the tasks in the list are always chronological"?

Comment (nicole-luo-exe) [reply docs/team/xiongjya.md:27]: Just a suggestion, current wording is fine as well!

File: docs/team/xiongjya.md [hunk 2]
```diff
@@ -14,35 +14,52 @@ and it still has the benefits of a Graphical User Interface (GUI) created with J
...
+    * Justification: As the user will be more likely to start on tasks that are due sooner, he/she will prefer to view such tasks in a more accessible manner. Thus, tasks that are due sooner are displayed at the top of the task list. 
+  
 
 * **New Feature**: Added a remind command that allows the user to view the tasks coming up in the next 3 days.
-   * What it does: Detects whether the tasks added to NurseyBook have dates that are scheduled for the next 3 days (in comparison to the time of running the command), and displays the result to the user.
-   * Justification: This feature similarly adds to the user experience. If a user has a task list cluttered up with multiple irrelevant and undeleted tasks, the command assists the user in detecting tasks of higher priority (in less than a second).
-   * Highlights: This implementation marks the start of incorporating the usage of time and duration into NurseyBook.
+    * What it does: Detects whether the tasks added to NurseyBook have dates that are scheduled for the next 3 days (in comparison to the time of running the command), and displays the result to the user.
```
Comment (nicole-luo-exe) [inline docs/team/xiongjya.md:32]: "are scheduled within" might be better?

**Review by nicole-luo-exe — CHANGES_REQUESTED** [review]
Looks fine! A few suggestions for wording but if you think it's alright just drop a reply and I'll approve.

File: docs/team/xiongjya.md (hunk 1 above)
Comment (xiongjya) [reply docs/team/xiongjya.md:27]: Alright! Such wording seems clearer too.

**Review by nicole-luo-exe — APPROVED** [review]
LGTM!

**Pull request merged**